src/lib/y2storage/disk_analyzer.rb

```diff
@@ -172,18 +172,29 @@ def data_for(*disks, data)
       disks.map { |d| @disks_data[data][d.name] }.flatten.compact
     end
 
-    # Obtains a list of disk devices.
+    # Obtains a list of disk devices, software RAIDs, and/or Bcaches
```

Comment:
NP: why "and/or" ?

Reply:
Hmmm... because the list could contain all of them (and) or whichever (or). Is it wrong? Do you prefer only "and"?

Reply:
I would use "and".